CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.5.7
+
+This is a maintenance release
+
+* [ORCH-2282](https://tickets.puppetlabs.com/browse/ORCH-2282) - Updates to current clj-parent
```

Comment:
Though we don't believe this to cause any regressions, or really provide new features, I'd make it a Y since so many dependencies are changing.

Reply:
Bumped to 1.6.0-SNAPSHOT.

Reply:
I know we had some confusion on our own side about how development of nrepl had moved form clojure/tools.nrepl to nrepl/nrepl -- I'm not sure how familiar users would be with that change, but as a newcomer I might appreciate a small note about that transition here. I wouldn't block on that though.

Reply:
Updated!